Pizza/Pizza/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace Pizza
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            //client
+            var director = new Director();
+            var builder = new WheatPizzaBuilder();

```

Comment:
It would be better if you used polymorphism to define builders, without creating new builder variable for each one.